From 7a6e8b70a956a722f90d551e920c1096a9455b54 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Sat, 5 Dec 2015 14:10:28 +0100
Subject: [PATCH 07/12] header: fix genl_notify() on kernel 4.4

The change of the parameters of genl_notify() is in kernel 4.4. This
backport should not be applied for kernel 4.4 and newer kernel versions.

Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 backport/backport-include/net/genetlink.h | 2 ++
 1 file changed, 2 insertions(+)

--- a/backport-include/net/genetlink.h
+++ b/backport-include/net/genetlink.h
@@ -146,10 +146,12 @@ backport_genlmsg_multicast_allns(struct
 #define __genl_const
 #else /* < 3.13 */
 #define __genl_const const
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,4,0)
 #define genl_notify(_fam, _skb, _info, _group, _flags)			\
 	genl_notify(_fam, _skb, genl_info_net(_info),			\
 		    genl_info_snd_portid(_info),			\
 		    _group, _info->nlhdr, _flags)
+#endif /* < 4.4 */
 #endif /* < 3.13 */
 
 #endif /* __BACKPORT_NET_GENETLINK_H */
